plugin/writeas-vim.vim

```diff
@@ -101,6 +105,10 @@ def _blogpost(title):
         v.current.buffer.append("posted: {} \n".format(output['data']['created']))
 EOF
 
-command! -nargs=1 AnonPost :python _anonpost(<f-args>)
-command! -nargs=1 BlogPost :python _blogpost(<f-args>)
-
+if has('python')
```

Comment:
Since you've already got the test above for `has('python')` versus python3, might want to set a string there that's `:python` or `:python3` and then use that here. No need to perform the check twice and you can just pass in the necessary dependency here.